### PR TITLE
Refactor/dom activity steps

### PR DESCRIPTION
**Pull request by e-spitz**
# Description

Refactor user steps activity card on DOM with API data

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [x] Refactoring

# How Has This Been Tested?
localhost8080

- [x] open index.html
- [x] dev tools
- [ ] terminal

# Checklist:

- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -74,177 +71,144 @@ let trendingStairsPhraseContainer = document.querySelector('.trending-stairs-phr
...
   dropdownGoal.innerText = `DAILY STEP GOAL | ${randomUser.dailyStepGoal}`;
   dropdownEmail.innerText = `EMAIL | ${randomUser.email}`;
   dropdownName.innerText = randomUser.name.toUpperCase();
   headerName.innerText = `${randomUser.getFirstName()}'S `;
-  updateTrendingStepDays(randomUser);
+  renderUserStepsActivity(randomUser, userRepo);
+  renderFriendStepActivity(userRepo);
+}
+
+const renderUserStepsActivity = (user, userRepo) => {
```
Comment (EricSergeant) [inline src/scripts.js:118]: Great refactoring and improvement to this function.

**Review by EricSergeant — APPROVED** [review]
Great refactoring and functionality implementation.

File: src/scripts.js [hunk 2]
```diff
@@ -74,177 +71,144 @@ let trendingStairsPhraseContainer = document.querySelector('.trending-stairs-phr
...
+  user.findTrendingStepDays();
+  trendingStepsPhraseContainer.innerHTML = `<p class='trend-line'>${user.trendingStepDays[0]}</p>`;
+  stepsCalendarTotalActiveMinutesWeekly.innerText = user.calculateAverageMinutesActiveThisWeek(todayDate)
+  stepsCalendarTotalStepsWeekly.innerText = user.calculateAverageStepsThisWeek(todayDate);
+}
+
+const renderFriendStepActivity = (userRepo) => {
+  stepsFriendActiveMinutesAverageToday.innerText = userRepo.calculateAverageMinutesActive(todayDate);
+  stepsFriendAverageStepGoal.innerText = userRepo.calculateAverageStepGoal();
+  stepsFriendStepsAverageToday.innerText = userRepo.calculateAverageSteps(todayDate);
```
Comment (dorifuto-dev) [inline src/scripts.js:131]: Good job refactoring and removing functionless statements

**Review by dorifuto-dev — APPROVED** [review]
Looks great! Good job getting the fetched data to display on the DOM.

**Pull request merged**